Automattic/vip-go-mu-plugins
# Remove Jetpack Search Menu if Enterprise Search is enabled

## Description

Jetpack 10 introduced a Jetpack Search marketing page in `wp-admin`. That page is not actionable if the client site has Enterprise Search enabled. 

<img width="1617" alt="Screen Shot 2021-09-10 at 10 12 37 AM" src="https://user-images.githubusercontent.com/7188409/132822313-327e8054-f588-43fc-b18d-0551ab87a3fb.png">

This PR removes that page if that's the case.

## Changelog Description

### Remove Jetpack Search Menu if Enterprise Search is enabled

We hide a Jetpack Search-related page if Enterprise Search is enabled.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test


1. Check out PR.
2. Make sure Jetpack and Enterprise Search are enabled.
3. You shouldn't be able to access the page above.
